relay_benchmark_test.go

```diff
@@ -223,3 +223,32 @@ func BenchmarkRelay2Servers5Clients4k(b *testing.B) {
 	p.servers = 2
 	benchmarkRelay(b, p)
 }
+
+func benchmarkRelayWithAppends(b *testing.B, n int) {
+	p := defaultParams()
+	for i := 0; i < n; i++ {
+		p.appends = append(p.appends, relay.KeyVal{Key: []byte("foo"), Val: []byte("bar")})
+	}
+	b.ResetTimer()
+	benchmarkRelay(b, p)
+}
+
+func BenchmarkRelayAppends0(b *testing.B) {
+	benchmarkRelayWithAppends(b, 0)
+}
+
+func BenchmarkRelayAppends1(b *testing.B) {
+	benchmarkRelayWithAppends(b, 1)
+}
+
+func BenchmarkRelayAppends2(b *testing.B) {
+	benchmarkRelayWithAppends(b, 2)
+}
+
+func BenchmarkRelayAppends5(b *testing.B) {
+	benchmarkRelayWithAppends(b, 5)
+}
+
+func BenchmarkRelayAppends10(b *testing.B) {
+	benchmarkRelayWithAppends(b, 10)
+}
```

Comment:
Use `b.Run` instead,
```
func BenchmarkRelayAppends(b *testing.B) {
  for _, appends := range []int{0, 1, 2, 5, 10} {
    b.Run(fmt.Sprintf("%v appends", appends), func(b *testing.B) {

    })
  }
}
```

no need for many functions, and can have the logic of `benchmarkRelayWithAppends` directly in the single function